### DISCUSSION TITLE
git push 推送不上去

### DISCUSSION THREAD

**Discussion by gentelyu** [community/community#88491] [Actions]
### Select Topic Area

Product Feedback

### Body

我和我身边的朋友都发现，在自己的vs code 上无法git push 文件到我的远程仓库。 在解除关联、重新建立映射也没有用，他显示fatal: The current branch master has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin master

我根据他的提示输入了这行指令也不行，终端一直没有反应

**Comment by ghostinhershell** [comment]
Hey there @gentelyu 👋🏾 

Thanks so much for reaching out in the Actions community. I do see that a similar question was answered before in our community. Can you check and see if the solution here also works for you?
- https://github.com/orgs/community/discussions/27305#discussioncomment-3281574

**Comment by Sodiumchloridy** [comment]
可以尝试：
1. git remote add origin {远程仓库的链接}
2. git push origin --all

May try:
1. git remote add origin {remote repository url}
2. git push origin --all
